### ISSUE TITLE
Support Adobe Reader Touch

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4008]
**Reported by supanut2000 on 2014-03-21 13:02**
Hi,
Could you please improve NVDA's support for metro apps on Windows 8 and 8.1 in the next or future release? Some apps, like Adobe Reader Touch, doesn't work with NVDA.

**Comment by bhavyashah** [comment]
@jcsteh Given that we have no means of contacting the original author of this ticket, do you think we have sufficient information to diagnose this issue or should we close it for the time being? Could the Github username of k_kolev1985 also be shared by someone and consequently mentioned?

**Comment by jcsteh** [comment]
The reporter suggests that no focus is reported at all in this app, but we don't know anything beyond that. It might be useful to have an advanced user take a look and see if they can figure anything out. That said, Adobe don't seem to be promoting this at all and Adobe Reader desktop is accessible for screen reader users, so I'm going to close this as won't fix. We can reopen if there is demand for this.
